packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -351,7 +351,7 @@ class SelectableRegionState extends State<SelectableRegion> with TextSelectionDe
   final LayerLink _startHandleLayerLink = LayerLink();
   final LayerLink _endHandleLayerLink = LayerLink();
   final LayerLink _toolbarLayerLink = LayerLink();
-  final _SelectableRegionContainerDelegate _selectionDelegate = _SelectableRegionContainerDelegate();
+  final MultiStaticSelectableSelectionContainerDelegate _selectionDelegate = MultiStaticSelectableSelectionContainerDelegate();
```

Comment:
Just wanted to check if this name would be any better if it was shorter. Maybe `MultiStaticSelectableDelegate` or `StaticSelectablesSelectionContainerDelegate`?

Reply:
Updated to `StaticSelectionContainerDelegate`, open to other suggestions. We have a private `_ScrollableSelectionContainerDelegate`, so I think this name makes sense given that convention.